documentation/docs/steps/apiKeyValueMapUpload.md

```diff
@@ -0,0 +1,45 @@
+# ${docGenStepName}
+
+## ${docGenDescription}
+
+With this step you can store one or more key value pairs of data stored in a group called a map or key value map.
+
+To consume the ApiKeyValueMapUpload step, proceed as follows:
+
+* Get the SAP API management service key from the SAP BTP sub account cockpit, under instance and subscriptions for service API Management, API portal under apiportal-apiaccess plan.
+* Store your service key created for SAP API Management in the the Jenkins server as a secret text.
+* Create a new Jenkins pipeline designated for the ApiKeyValueMapUpload step.
+* Execute the pipeline and validate step exection results as explained in the blog [Integration Suite Piper commands](https://blogs.sap.com/2022/01/05/orking-with-integration-suite-piper-commands/)
+
+* Using ApiKeyValueMapUpload step, you can create new API key value map in the API portal.
+* The ApiKeyValueMapUpload step allows you to prevent command execution in case the/an API key value map already exists.
+* If API key value map already exists, then delete it and execute the piper step again, which create new API Key value Map.
+* ApiKeyValueMapUpload only supports create operation.
```

Comment:
So you did not want to reference the other steps that can do the other operations? Is it because the other operations are not supported at all yet in Piper?

Reply:
yes correct